[Python][Flaky Test] test_sync_standalone_scan_with_count

### Test Name

tests/sync_tests/test_sync_scan.py::TestSyncScan::test_sync_standalone_scan_with_count

### Test Location

/valkey-glide/python/tests/sync_tests/test_sync_scan.py: 451

### Failure Permlink

https://github.com/valkey-io/valkey-glide/actions/runs/21383486312/job/61555004313#step:9:6982

### System Information

aarch64-apple-darwin

### Logs
```
tests/sync_tests/test_sync_scan.py::TestSyncScan::test_sync_standalone_scan_with_count[ProtocolVersion.RESP2-False]
~~~~~~~~~~~~~~~~ Stack of ThreadPoolExecutor-7_0 (14177169408) ~~~~~~~~~~~~~~~~~
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 1032, in _bootstrap
    self._bootstrap_inner()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 1075, in _bootstrap_inner
    self.run()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 1012, in run
    self._target(*self._args, **self._kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/concurrent/futures/thread.py", line 93, in _worker
    work_item.run()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/concurrent/futures/thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/Users/admin/actions-runner/_work/valkey-glide/valkey-glide/python/tests/sync_tests/test_sync_client.py", line 1489, in endless_blmpop_call
    glide_sync_client.blmpop([key3], ListDirection.LEFT, 0, 1)
  File "/Users/admin/actions-runner/_work/valkey-glide/valkey-glide/python/.env/lib/python3.12/site-packages/glide_sync/sync_commands/core.py", line 1860, in blmpop
    self._execute_command(RequestType.BLMPop, args),
  File "/Users/admin/actions-runner/_work/valkey-glide/valkey-glide/python/.env/lib/python3.12/site-packages/glide_sync/glide_client.py", line 423, in _execute_command
    result = self._lib.command(
~~~~~~~~~~~~~~~~ Stack of ThreadPoolExecutor-6_0 (14160343040) ~~~~~~~~~~~~~~~~~
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 1032, in _bootstrap
    self._bootstrap_inner()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 1075, in _bootstrap_inner
    self.run()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 1012, in run
    self._target(*self._args, **self._kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/concurrent/futures/thread.py", line 93, in _worker
    work_item.run()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/concurrent/futures/thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/Users/admin/actions-runner/_work/valkey-glide/valkey-glide/python/tests/sync_tests/test_sync_client.py", line 1489, in endless_blmpop_call
    glide_sync_client.blmpop([key3], ListDirection.LEFT, 0, 1)
  File "/Users/admin/actions-runner/_work/valkey-glide/valkey-glide/python/.env/lib/python3.12/site-packages/glide_sync/sync_commands/core.py", line 1860, in blmpop
    self._execute_command(RequestType.BLMPop, args),
  File "/Users/admin/actions-runner/_work/valkey-glide/valkey-glide/python/.env/lib/python3.12/site-packages/glide_sync/glide_client.py", line 423, in _execute_command
    result = self._lib.command(
~~~~~~~~~~~~~~~~ Stack of ThreadPoolExecutor-5_0 (14143516672) ~~~~~~~~~~~~~~~~~
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 1032, in _bootstrap
    self._bootstrap_inner()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 1075, in _bootstrap_inner
    self.run()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 1012, in run
    self._target(*self._args, **self._kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/concurrent/futures/thread.py", line 93, in _worker
    work_item.run()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/concurrent/futures/thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/Users/admin/actions-runner/_work/valkey-glide/valkey-glide/python/tests/sync_tests/test_sync_client.py", line 1489, in endless_blmpop_call
    glide_sync_client.blmpop([key3], ListDirection.LEFT, 0, 1)
  File "/Users/admin/actions-runner/_work/valkey-glide/valkey-glide/python/.env/lib/python3.12/site-packages/glide_sync/sync_commands/core.py", line 1860, in blmpop
    self._execute_command(RequestType.BLMPop, args),
  File "/Users/admin/actions-runner/_work/valkey-glide/valkey-glide/python/.env/lib/python3.12/site-packages/glide_sync/glide_client.py", line 423, in _execute_command
    result = self._lib.command(
~~~~~~~~~~~~~~~~ Stack of ThreadPoolExecutor-4_0 (14126690304) ~~~~~~~~~~~~~~~~~
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 1032, in _bootstrap
    self._bootstrap_inner()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 1075, in _bootstrap_inner
    self.run()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 1012, in run
    self._target(*self._args, **self._kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/concurrent/futures/thread.py", line 93, in _worker
    work_item.run()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/concurrent/futures/thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/Users/admin/actions-runner/_work/valkey-glide/valkey-glide/python/tests/sync_tests/test_sync_client.py", line 1489, in endless_blmpop_call
    glide_sync_client.blmpop([key3], ListDirection.LEFT, 0, 1)
  File "/Users/admin/actions-runner/_work/valkey-glide/valkey-glide/python/.env/lib/python3.12/site-packages/glide_sync/sync_commands/core.py", line 1860, in blmpop
    self._execute_command(RequestType.BLMPop, args),
  File "/Users/admin/actions-runner/_work/valkey-glide/valkey-glide/python/.env/lib/python3.12/site-packages/glide_sync/glide_client.py", line 423, in _execute_command
    result = self._lib.command(
~~~~~~~~~~~~~~~~ Stack of ThreadPoolExecutor-3_0 (14109863936) ~~~~~~~~~~~~~~~~~
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 1032, in _bootstrap
    self._bootstrap_inner()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 1075, in _bootstrap_inner
    self.run()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 1012, in run
    self._target(*self._args, **self._kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/concurrent/futures/thread.py", line 93, in _worker
    work_item.run()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/concurrent/futures/thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/Users/admin/actions-runner/_work/valkey-glide/valkey-glide/python/tests/sync_tests/test_sync_client.py", line 1404, in endless_blpop_call
    glide_sync_client.blpop(["non_existent_key"], 0)
  File "/Users/admin/actions-runner/_work/valkey-glide/valkey-glide/python/.env/lib/python3.12/site-packages/glide_sync/sync_commands/core.py", line 1769, in blpop
    self._execute_command(RequestType.BLPop, keys + [str(timeout)]),
  File "/Users/admin/actions-runner/_work/valkey-glide/valkey-glide/python/.env/lib/python3.12/site-packages/glide_sync/glide_client.py", line 423, in _execute_command
    result = self._lib.command(
~~~~~~~~~~~~~~~~~ Stack of ThreadPoolExecutor-2_0 (6439628800) ~~~~~~~~~~~~~~~~~
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 1032, in _bootstrap
    self._bootstrap_inner()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 1075, in _bootstrap_inner
    self.run()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 1012, in run
    self._target(*self._args, **self._kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/concurrent/futures/thread.py", line 93, in _worker
    work_item.run()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/concurrent/futures/thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/Users/admin/actions-runner/_work/valkey-glide/valkey-glide/python/tests/sync_tests/test_sync_client.py", line 1404, in endless_blpop_call
    glide_sync_client.blpop(["non_existent_key"], 0)
  File "/Users/admin/actions-runner/_work/valkey-glide/valkey-glide/python/.env/lib/python3.12/site-packages/glide_sync/sync_commands/core.py", line 1769, in blpop
    self._execute_command(RequestType.BLPop, keys + [str(timeout)]),
  File "/Users/admin/actions-runner/_work/valkey-glide/valkey-glide/python/.env/lib/python3.12/site-packages/glide_sync/glide_client.py", line 423, in _execute_command
    result = self._lib.command(
~~~~~~~~~~~~~~~~~ Stack of ThreadPoolExecutor-1_0 (6422802432) ~~~~~~~~~~~~~~~~~
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 1032, in _bootstrap
    self._bootstrap_inner()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 1075, in _bootstrap_inner
    self.run()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 1012, in run
    self._target(*self._args, **self._kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/concurrent/futures/thread.py", line 93, in _worker
    work_item.run()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/concurrent/futures/thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/Users/admin/actions-runner/_work/valkey-glide/valkey-glide/python/tests/sync_tests/test_sync_client.py", line 1404, in endless_blpop_call
    glide_sync_client.blpop(["non_existent_key"], 0)
  File "/Users/admin/actions-runner/_work/valkey-glide/valkey-glide/python/.env/lib/python3.12/site-packages/glide_sync/sync_commands/core.py", line 1769, in blpop
    self._execute_command(RequestType.BLPop, keys + [str(timeout)]),
  File "/Users/admin/actions-runner/_work/valkey-glide/valkey-glide/python/.env/lib/python3.12/site-packages/glide_sync/glide_client.py", line 423, in _execute_command
    result = self._lib.command(
~~~~~~~~~~~~~~~~~ Stack of ThreadPoolExecutor-0_0 (6405976064) ~~~~~~~~~~~~~~~~~
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 1032, in _bootstrap
    self._bootstrap_inner()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 1075, in _bootstrap_inner
    self.run()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/threading.py", line 1012, in run
    self._target(*self._args, **self._kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/concurrent/futures/thread.py", line 93, in _worker
    work_item.run()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/concurrent/futures/thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/Users/admin/actions-runner/_work/valkey-glide/valkey-glide/python/tests/sync_tests/test_sync_client.py", line 1404, in endless_blpop_call
    glide_sync_client.blpop(["non_existent_key"], 0)
  File "/Users/admin/actions-runner/_work/valkey-glide/valkey-glide/python/.env/lib/python3.12/site-packages/glide_sync/sync_commands/core.py", line 1769, in blpop
    self._execute_command(RequestType.BLPop, keys + [str(timeout)]),
  File "/Users/admin/actions-runner/_work/valkey-glide/valkey-glide/python/.env/lib/python3.12/site-packages/glide_sync/glide_client.py", line 423, in _execute_command
    result = self._lib.command(
+++++++++++++++++++++++++++++++++++ Timeout ++++++++++++++++++++++++++++++++++++

FAILED tests/sync_tests/test_sync_scan.py::TestSyncScan::test_sync_standalone_scan_with_count[ProtocolVersion.RESP2-False] - Failed: Timeout (>300.0s) from pytest-timeout.
```

### Language and Version

Python 3.12

### Engine Version

Valkey 7.0

### Glide Version

main

--- a/python/tests/sync_tests/test_sync_client.py
+++ b/python/tests/sync_tests/test_sync_client.py
@@ -1526,9 +1526,9 @@ class TestCommands:
             glide_sync_client.blpop(["foo"], 0.001)
 
         def endless_blpop_call():
-            glide_sync_client.blpop(["non_existent_key"], 0)
+            glide_sync_client.blpop(["non_existent_key"], 10)
 
-        # blpop is called against a non-existing key with no timeout, but we wrap the call in the `run_sync_func_with_timeout_in_thread` function
+        # blpop is called against a non-existing key with a long timeout, but we wrap the call in the `run_sync_func_with_timeout_in_thread` function
         # to avoid having the test block forever
         with pytest.raises(TimeoutError):
             run_sync_func_with_timeout_in_thread(
@@ -1608,10 +1608,10 @@ class TestCommands:
         with pytest.raises(RequestError):
             glide_sync_client.blmpop([key4], ListDirection.LEFT, 0.1, 1)
 
-        # BLMPOP is called against a non-existing key with no timeout, but we wrap the call in a timeout to
+        # BLMPOP is called against a non-existing key with a long timeout, but we wrap the call in a timeout to
         # avoid having the test block forever
         def endless_blmpop_call():
-            glide_sync_client.blmpop([key3], ListDirection.LEFT, 0, 1)
+            glide_sync_client.blmpop([key3], ListDirection.LEFT, 10, 1)
 
         with pytest.raises(TimeoutError):
             run_sync_func_with_timeout_in_thread(
@@ -1705,9 +1705,9 @@ class TestCommands:
             glide_sync_client.brpop(["foo"], 0.001)
 
         def endless_brpop_call():
-            glide_sync_client.brpop(["non_existent_key"], 0)
+            glide_sync_client.brpop(["non_existent_key"], 10)
 
-        # brpop is called against a non-existing key with no timeout, but we wrap the call in the `run with timeout` function
+        # brpop is called against a non-existing key with a long timeout, but we wrap the call in the `run with timeout` function
         # to avoid having the test block forever
         with pytest.raises(TimeoutError):
             run_sync_func_with_timeout_in_thread(
@@ -1916,7 +1916,7 @@ class TestCommands:
                 key1, key3, ListDirection.LEFT, ListDirection.LEFT, 0.1
             )
 
-        # BLMOVE is called against a non-existing key with no timeout, but we wrap the call in a timeout to
+        # BLMOVE is called against a non-existing key with a long timeout, but we wrap the call in a timeout to
         # avoid having the test block forever
         def endless_blmove_call():
             glide_sync_client.blmove(
@@ -1924,7 +1924,7 @@ class TestCommands:
                 key2,
                 ListDirection.LEFT,
                 ListDirection.RIGHT,
-                0,
+                10,
             )
 
         with pytest.raises(TimeoutError):
@@ -4128,9 +4128,9 @@ class TestCommands:
             glide_sync_client.bzpopmin(["foo"], 0.5)
 
         def endless_bzpopmin_call():
-            glide_sync_client.bzpopmin(["non_existent_key"], 0)
+            glide_sync_client.bzpopmin(["non_existent_key"], 10)
 
-        # bzpopmin is called against a non-existing key with no timeout, but we wrap the call the `run_sync_func_with_timeout_in_thread` function
+        # bzpopmin is called against a non-existing key with a long timeout, but we wrap the call the `run_sync_func_with_timeout_in_thread` function
         # to avoid having the test block forever
         with pytest.raises(TimeoutError):
             run_sync_func_with_timeout_in_thread(
@@ -4189,9 +4189,9 @@ class TestCommands:
             glide_sync_client.bzpopmax(["foo"], 0.5)
 
         def endless_bzpopmax_call():
-            glide_sync_client.bzpopmax(["non_existent_key"], 0)
+            glide_sync_client.bzpopmax(["non_existent_key"], 10)
 
-        # bzpopmax is called against a non-existing key with no timeout, but we wrap the call in the `run_sync_func_with_timeout_in_thread` function
+        # bzpopmax is called against a non-existing key with a long timeout, but we wrap the call in the `run_sync_func_with_timeout_in_thread` function
         # to avoid having the test block forever
         with pytest.raises(TimeoutError):
             run_sync_func_with_timeout_in_thread(
@@ -4866,9 +4866,9 @@ class TestCommands:
         assert compare_maps(entries, result_map) is True  # type: ignore
 
         def endless_bzmpop_call():
-            glide_sync_client.bzmpop(["non_existent_key"], ScoreFilter.MAX, 0)
+            glide_sync_client.bzmpop(["non_existent_key"], ScoreFilter.MAX, 10)
 
-        # bzmpop is called against a non-existing key with no timeout, but we wrap the call in the `run_sync_func_with_timeout_in_thread` function
+        # bzmpop is called against a non-existing key with a long timeout, but we wrap the call in the `run_sync_func_with_timeout_in_thread` function
         # to avoid having the test block forever
         with pytest.raises(TimeoutError):
             run_sync_func_with_timeout_in_thread(

